src/main/java/org/prebid/server/bidder/dmx/DmxBidder.java

```diff
@@ -51,6 +54,7 @@
     private static final int SECURE = 1;
     private static final String IMP = "</Impression><Impression><![CDATA[%s]]></Impression>";
     private static final String SEARCH = "</Impression>";
+    private static final List<Integer> VIDEO_PROTOCOLS = new ArrayList<>(Arrays.asList(2, 3, 5, 6, 7, 8));
```

Comment:
Set seems to be better data type
``` java
    private static final Set<Integer> PROTOCOLS = IntStream.range(2, 9).boxed().collect(Collectors.toSet());
```

Reply:
That won't work there. We have `2, 3, 5, 6, 7, 8`, not `2, 3, 4, 5, 6, 7, 8`. So should be either skipping 4, either going without `IntStream`

Reply:
Agree